pkg/scheduler/framework/session.go

```diff
@@ -127,8 +126,10 @@ func openSession(cache cache.Cache) *Session {
 
 		TotalResource:  api.EmptyResource(),
 		TotalGuarantee: api.EmptyResource(),
-		podGroupStatus: map[api.JobID]scheduling.PodGroupStatus{},
-
+		PodGroupCache: api.PodGroupCache{
```

Comment:
It is better to rename as `PodGroupOldState`, which is used to record some old PodGroup data.

Reply:
done